### PR TITLE
How to install neovim on ubuntu 1804

### PR DESCRIPTION
**Pull request by hubvu**
Adding a tutorial on how to install Neovim on Ubuntu Desktop 18.04 LTS.

### CODE REVIEW

File: tutorials/desktop/how-to-install-neovim-on-ubuntu-1804/how-to-install-neovim-on-ubuntu-1804.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+* Type into the terminal, `nvim` and press the `Enter` key to continue.
+* Click on the "Apps" icon, search for `neovim` and press the `Enter` key to continue, as demonstrated below;
+
+![Neovim Startup](./images/04-neovim-start.png)
+
+A welcome window should appear displaying information about the version of Neovim you are currently running on and helpful information to get started with the text editor.
+
+![Neovim Welcome](./images/05-neovim-welcome-window.png)
+
+### Basic usage
```
Comment (degville) [inline tutorials/desktop/how-to-install-neovim-on-ubuntu-1804/how-to-install-neovim-on-ubuntu-1804.md]: I think this subheading should be something different to the title for this step. Also, a little more on using Neovim may be helpful, even if it's just something on navigation and insert and command modes.

**Review by degville — COMMENTED** [review]
Congratulations on your first tutorial! Thanks so much for sticking to the guidelines and writing so clearly.

My only real overall comment is that it would be great if the tutorial shared a little more insight than simply installing and running something, especially a text editor favoured by more technical users, but we can always update to add further details. 

I've also been a little overly pedantic with my comments, but only to try and share our perspective when writing tutorials for as wide an audience as possible. Generally, I think the tutorial is a great start; it's clear and concise and does what it promises with no real mistakes. And that's not common! Thanks!

**Comment by hubvu** [comment]
- Thanks for the great feedback, I wondered earlier if I should add more information on basic usage but then I thought that would be a good separate tutorial instead, but I'll take your advice and include a walk through of basic vim commands and notable neovim features.

**Comment by hubvu** [comment]
- I will make the changes requested when I have some idle time and resubmit.

**Comment by steverydz** [comment]
Thanks for submitting a PR for this tutorial. We have moved our tutorials to discourse. To contribute going forwards please go to https://discourse.ubuntu.com/c/tutorials.

**Pull request closed without merging**